README.md

```diff
@@ -67,7 +67,6 @@ minio.exe server D:\Photos
 chmod 755 minio
 ./minio server ~/Photos
 ```
-Please visit official zfs FreeBSD guide for more details [here](https://www.freebsd.org/doc/handbook/zfs-quickstart.html)
```

Comment:
Point to this instead - https://docs.minio.io/docs/how-to-run-minio-in-freenas for further exploration.